build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("com.diffplug.gradle.spotless")
     id("com.github.ben-manes.versions") version "0.27.0"
+    id("org.sonarqube") version "2.8"
```

Comment:
the recent error seems to be connected with this scanner version maybe 3.0 would be a better fit based on https://plugins.gradle.org/plugin/org.sonarqube

link for an issue description https://community.sonarsource.com/t/azure-devops-build-with-sonarcloud-error/16787/5